src/scripts/actions/UploaderActions.js

```diff
@@ -33,6 +33,8 @@ const transcodingProgress = createAction(TRANSCODING_PROGRESS)
 const transcodingSuccess = createAction(TRANSCODING_SUCCESS)
 const transcodingFailure = createAction(TRANSCODING_FAILURE)
 
+console.log(store)
```

Comment:
What is this needed for? (if you are trying to inspect the redux store I _highly_ recommend using the `redux devtools` chrome extension

Reply:
https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en